internal/core/operations/deletescheduler/delete_scheduler_executor_test.go

```diff
@@ -265,7 +264,7 @@ func TestDeleteSchedulerExecutor_Execute(t *testing.T) {
 
 			err := executor.Execute(ctx, op, definition)
 
-			require.Equal(t, operations.NewErrUnexpected(errors.New("some error on storage")), err)
+			require.Equal(t, errors.New("some error on storage"), err)
```

Comment:
What do you think to use here `require.EqualError` ([link](https://pkg.go.dev/github.com/stretchr/testify/require#EqualError)) and instead of creating a new error, you will only pass the error message as in the link.